app/backend/load_azd_env.py

```diff
@@ -0,0 +1,23 @@
+import json
+import logging
+import subprocess
+
+from dotenv import load_dotenv
+
+logger = logging.getLogger("scripts")
+
+
+def load_azd_env():
```

Comment:
This file actually shows up in two places, both here and in scripts folder, for convenience.